api/MMRProject.Api/DTOs/SeasonDto.cs

```diff
@@ -0,0 +1,11 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace MMRProject.Api.DTOs;
+
+public record SeasonDto
+{
+    [Required]
+    public required long Id { get; set; }
+
+    public DateTime? CreatedAt { get; set; }
```

Comment:
Can we determine the start+end of a season? And show that in the UI?

Reply:
I mean yes, kinda. But once we have to paginate the solution I see breaks.

Reply:
Actually. Can't think of a good solution at the moment. So maybe future work.